VirtoCommerce.Platform.Security/Services/LimitedPermissionsHandler.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.Platform.Security.Services
+{
+    public class LimitedPermissionsHandler
```

Comment:
Could you explain why we need this handler for this simple checks? Why we can't do this extra checks  in PermissionAuthorizationHandler?

Reply:
We need it to avoid duplicating. It is using in the HangfireAuthorizationHandler also.